lib/compat/wordpress-6.2/get-global-styles-and-settings.php

```diff
@@ -1,87 +0,0 @@
-<?php
-/**
- * API to interact with global settings & styles.
- *
- * @package gutenberg
- */
-
-if ( ! function_exists( 'wp_theme_has_theme_json' ) ) {
```

Comment:
Removing this file is ok. It was introduced in 6.2, and it was already using the core functions, when available.